cmd/topology/main.go

```diff
@@ -50,10 +51,21 @@ func main() {
 		keyFile = defaultKeyFile
 	}
 
+	var bindPort int
+	portEnv := os.Getenv("PORT")
+	if portEnv != "" {
```

Comment:
What is the port value if PORT=""

Reply:
In that case, bindPort should be init'ed to 0 and in internal/service/service.go, this value means to use the default, which would be 443.